test/runner/compact_reporter_test.dart

```diff
@@ -163,6 +163,15 @@ void main() {
         +0 -1: Some tests failed.""");
   });
 
+  test("Respects the --line-length argument", () {
+    return _expectReport("""
+        test('A test with a pretty long name', () {});""", """
```

Comment:
Nit: Since this `test()` invocation is just one line, there's no need to make it a multiline string.

Reply:
done